src/routers/datatypeRouters/edges/diseases_genes.ts

```diff
@@ -90,9 +90,22 @@ async function conditionalGeneSearch (input: paramsFormatType): Promise<any[]> {
   return await router.getSources(input, '_key', input.verbose === 'true', edgeQuery(input))
 }
 
+const geneQuery = genesQueryFormat.omit({
+  organism: true,
+  name: true
+}).merge(z.object({
+  gene_name: z.string().trim().optional(),
+  source: z.string().trim().optional(),
+  page: z.number().default(0),
+  verbose: z.enum(['true', 'false']).default('false')
+})).merge(associationTypes).transform(({gene_name, ...rest}) => ({
+  name: gene_name,
+  ...rest
+}))
```

Comment:
the original `genesQueryFormat` has the updated `name` property from genes.
in any edge endpoint where we import the gene query object, we would have the `name` property, which is confusing in edge endpoints where more than one node is involved.
So here, we import the original query format, rename it to `gene_name` for the endpoint, and then rename the property back to `name`